sources/south-america/br/GeoPortal_DF_2016_imagery.geojson

```diff
@@ -11,7 +11,6 @@
         "available_projections": [
             "CRS:84",
             "EPSG:3857",
-            "EPSG:4326",
```

Comment:
`Could not open received data as image (Recieved filetype: empty` this seems to be an intermittent issue